smoke-tests/src/combined/java/com/google/firebase/testing/combined/AllTests.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Google LLC
```

Comment:
2019?

Reply:
I copied the header over from other files with the older date. I can update all of them in another change,